cmd/snap/wait.go

```diff
@@ -115,17 +117,38 @@ func (wmx waitMixin) wait(id string) (*client.Change, error) {
 			tMax = time.Time{}
 		}
 
+		maybeShowLog := func(t *client.Task) {
+			nowLog := lastLogStr(t.Log)
+			if lastLog[t.ID] != nowLog {
+				pb.Notify(nowLog)
+				lastLog[t.ID] = nowLog
+			}
+		}
+
+		// Tasks in "wait" state communicate the wait reason
+		// via the log mechanism. So make sure the log is
+		// visible even if the normal progress reporting
+		// has tasks in "Doing" state (like "check-refresh")
+		// that would suppress displaying the log. This will
+		// ensure on a classic+modes system the user sees
+		// the messages: "Task set to wait until a manual system restart allows to continue"
+		for _, t := range chg.Tasks {
+			if t.Status == "Wait" {
+				maybeShowLog(t)
+				waitCtrlcMsg.Do(func() {
+					fmt.Fprintf(Stdout, i18n.G("Warning: pressing ctrl-c will abort the running change\n"))
```

Comment:
we usually send warninga to stderr, no? They also usually start with WARNING and indeed most ends with .